src/radio/index.js

```diff
@@ -150,7 +155,10 @@ class Radio extends EventEmitter {
     const ratio = skipRatio(this.current.duration);
     const total = this.order.length;
     const needed = Math.ceil(ratio * total);
-    this.emit('skips', this.skips, needed);
+    this.emit('skips', this.skips);
```

Comment:
i think this should be part of `voteSkip()` instead

the only question is making sure we're correctly handling cases that used to directly call `checkSkips()`